docs/local-development.md

```diff
@@ -257,6 +259,58 @@ To review the concise and clear overview of the pipeline run, use the tkn cli.
 tkn pr describe bare-build-pipeline-run
 ```
 
+The output from above should look something like this:
+```
+Name:              bare-build-pipeline-run
+Namespace:         default
+Pipeline Ref:      python-build-pipeline
```

Comment:
Maybe we can condense this here:

The output from the above should look something like this (abbreviated):

```
Name:              bare-build-pipeline-run
Namespace:         default
Pipeline Ref:      python-build-pipeline
...
 ∙ bare-build-pipeline-run-list-dependencies      list-dependencies      6 hours ago   6 seconds    Succeeded
 ∙ bare-build-pipeline-run-install-dependencies   install-dependencies   6 hours ago   13 seconds   Succeeded
 ∙ bare-build-pipeline-run-fetch-from-git         fetch-from-git         6 hours ago   8 seconds    Succeeded
 ```